tests/containers/docker_image.pm

```diff
@@ -35,11 +35,6 @@ sub run {
 
     install_docker_when_needed();
 
-    if (is_sle()) {
```

Comment:
I like the idea, but we need to be careful. This test is also executed in other places, like in [JeOS](https://openqa.suse.de/tests/4806721#step/docker_image/12).

Reply:
In case of Jeos, there is no danger, cause the certificates are there. So, this should be safe.

Reply:
i triggered VR to test it.

Reply:
https://openqa.opensuse.org/tests/1433315#step/podman/173 fails but it is not relative to the the recent change as it seems to fail in the module before. i will have to try again when the other problem resolved